packages/bpk-stylesheets/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Backpack's stylesheets.",
   "main": "index.js",
   "scripts": {
-    "build": "webpack"
+    "build": "node build.js"
```

Comment:
We have to manually call `webpack` from `build.js` to avoid this error:

<img width="1165" alt="Screen Shot 2019-06-26 at 16 37 12" src="https://user-images.githubusercontent.com/73652/60194127-ab38f680-9830-11e9-8c44-d51ab7d01b16.png">